src/files-and-uploads/FileMenu.jsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import {
+  Dropdown,
+  IconButton,
+  Icon,
+} from '@edx/paragon';
+import messages from './messages';
+
+const FileMenu = ({
+  externalUrl,
+  handleDelete,
+  handleLock,
+  locked,
+  openAssetInfo,
+  portableUrl,
+  iconSrc,
+  id,
+  // injected
+  intl,
+}) => (
+  <Dropdown data-testid={`file-menu-dropdown-${id}`}>
+    <Dropdown.Toggle
+      id={`file-menu-dropdown-${id}`}
+      as={IconButton}
+      src={iconSrc}
+      iconAs={Icon}
+      variant="primary"
+      alt="asset-menu-toggle"
+    />
+    <Dropdown.Menu>
+      <Dropdown.Item
+        onClick={() => navigator.clipboard.writeText(portableUrl)}
+      >
+        {intl.formatMessage(messages.copyStudioUrlTitle)}
+      </Dropdown.Item>
+      <Dropdown.Item
+        onClick={() => navigator.clipboard.writeText(externalUrl)}
```

Comment:
I'm sure you're aware of how to get these lines covered. I don't think you have to.